src/io/common.jl

```diff
@@ -1,8 +1,18 @@
 function parse_file(file)
     if endswith(file, ".m")
-        return PowerModels.parse_matpower(file)
+        pm_data = PowerModels.parse_matpower(file)
     else
-        network_data = PowerModels.parse_json(file)
-        return network_data
+        pm_data = PowerModels.parse_json(file)
     end
-end
\ No newline at end of file
+
+    check_network_data(pm_data)
```

Comment:
should `check_network_data` happen inside `parse_matpower`, `parse_json`, etc? It modifies `pm_data` (see my comments in #95), and it feels weird to have `parse_matpower`, `parse_json`, etc return something different from `parse_file`.

Reply:
There are some special cases when one would not want to run `check_network_data`, but in the novice user case, it should always be run by default, if the provide a a file name or raw string data.